### PR TITLE
fixed shebang and path

### PR DESCRIPTION
**Pull request by kunalkushwaha**
Signed-off-by: Kunal Kushwaha <kushwaha_kunal_v7@lab.ntt.co.jp>

**What this PR does / why we need it**:
`make cli` reports to unable to find source as `path` was not. 
- Added shebang to fix this.
- made inline with other scripts in cluster/*.sh

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by davidvossel** [comment]
@kunalkushwaha Can you provide some more information about the steps we need to go through to reproduce what you're experiencing? ```make cli``` is not one of our make targets, so it is unclear to me what you've encountered.

**Comment by kunalkushwaha** [comment]
@davidvossel I was following the steps as specified in [dev environment docs](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#accessing-the-containerized-nodes-via-ssh).

Since, there was no shebang specified on top of file, `source: not found` error was reported.

again for `hack/common.sh`, `` source $(dirname "$0")/../hack/common.sh`` is not requored where in same file, ``source hack/config.sh`` is also specified.

It is basically simplied version as it is done in rest of file like cluster-up.sh etc

File: cluster/cli.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
-source $(dirname "$0")/../hack/common.sh
+#!/bin/bash
+set -e
 
+source hack/common.sh
 test -t 1 && USE_TTY="-it"
-
-source ${KUBEVIRT_DIR}/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source cluster/$KUBEVIRT_PROVIDER/provider.sh
```
Comment (davidvossel) [inline cluster/cli.sh]: I believe we have usecases where the KUBEVIRT_DIR environment variable is overwritten. Everything else in this patch looks okay though. Can we revert this line?

**Comment by kunalkushwaha** [comment]
Done. All test have passed too.

**Comment by davidvossel** [comment]
ci test please

**Pull request merged**